image_width: 640
image_height: 360
camera_name: narrow_stereo
camera_matrix:
  rows: 3
  cols: 3
  data: [266.082019, 0.000000, 317.509713, 0.000000, 265.623549, 179.159722, 0.000000, 0.000000, 1.000000]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [0.002098, -0.001889, 0.000012, 0.000250, 0.000000]
rectification_matrix:
  rows: 3
  cols: 3
  data: [1.000000, 0.000000, 0.000000, 0.000000, 1.000000, 0.000000, 0.000000, 0.000000, 1.000000]
projection_matrix:
  rows: 3
  cols: 4
  data: [265.423279, 0.000000, 317.281071, 0.000000, 0.000000, 265.083130, 178.666968, 0.000000, 0.000000, 0.000000, 1.000000, 0.000000]
